src/EnergyPlus/ScheduleManager.cc

```diff
@@ -2776,7 +2765,13 @@ namespace ScheduleManager {
         //  so, current date, but maybe TimeStep added
 
         // Hourly Value
-        int thisHour = ThisHour + state.dataEnvrn->DSTIndicator;
+        int thisHour;
+        if (state.dataScheduleMgr->Schedule(ScheduleIndex).UseDaylightSaving) {
+            thisHour = ThisHour + state.dataEnvrn->DSTIndicator;
+        } else {
+            thisHour = ThisHour;
+        }
+
```

Comment:
If the "UseDaylightSaving" boolean has a "true" value, the "thisHour" variable adds the DSTIndicator (value of 1 hour) to the "ThisHour" variable (note the capital vs small lettering in the variable names), otherwise, the hour value is unchanged (i.e., daylight savings time effect is not accounted for).

Reply:
I think Yoda taught me that booleans are simply 0 or 1, so I think this entire if/else could be reduced to
`int const thisHour = ThisHour + state.dataEnvrn->DSTIndicator * state.dataScheduleMgr->Schedule(ScheduleIndex).UseDaylightSaving;`
@Myoldmopar would this be the best solution?